README.md

```diff
@@ -25,6 +25,17 @@ const entity = entity('User', {
 
 const gql = entity2type(entity)
 ```
+If you put custom name or custom name or convention:
```

Comment:
will be interesting if we warn the developer to don't use both options at the same time

Reply:
in the new session it becomes clear